src/bessel.jl

```diff
@@ -590,6 +590,27 @@ function bessely(n::Integer, x::BigFloat)
     return z
 end
 
+softinvsqrt(x)          = sqrt(x)/(abs(x) + (eps ∘ float      )(x))
+softinvsqrt(z::Complex) = sqrt(z)/(abs(z) + (eps ∘ float ∘ abs)(z))
```

Comment:
```suggestion
softinvsqrt(z::Complex{T}) where {T} = sqrt(z) / (abs(z) + eps(float(T))))
```

Reply:
Wouldn't it make more sense to use `eps(float(z))` instead of `eps(Float64)` since this gives the epsilon *at* `z`, not the epsilon at `1.0`?

Reply:
Regardless, I switched to a conditional to just check if `x` is near zero instead of using `softinvsqrt`.